modules/imgproc/include/opencv2/imgproc/imgproc_c.h

```diff
@@ -934,6 +934,7 @@ CVAPI(void)  cvGoodFeaturesToTrack( const CvArr* image, CvArr* eig_image,
                                     double  min_distance,
                                     const CvArr* mask CV_DEFAULT(NULL),
                                     int block_size CV_DEFAULT(3),
+                                    int gradiant_size CV_DEFAULT(3),
```

Comment:
C API deprecated, so it's better to leave it as is.

Reply:
Moreover, it breaks ABI compatibility, so we don't need this